src/components/todoList/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+export const TodoList = ({ todo, deleteTodo, updateTodo }) => {
+  const [checked, setChecked] = useState(true);
+  const [newName, setNewName] = useState(todo.name);
+  const [editing, setEditing] = useState(false);
+
+  const toggleCheckBox = () => {
+    setChecked((prevState) => !prevState);
+    todo.completed = checked;
```

Comment:
props are immutable meaning they can't be altered inside children component, you need to callback a function inside the parent file i.e. index.js to update the completed status